styles/scss/pivotTable.scss

```diff
@@ -153,6 +153,20 @@ $header-cell-resize-width: 20px;
             border-top: 1px solid transparentize($gd-color-disabled, 0.5);
         }
     }
+
+    .gd-column-merged {
+        display: none;
+    }
+
+    .gd-cell-hide {
+        color: transparent;
+        border-top: 1px solid white;
+        background-color: white;
```

Comment:
Do we need to override bg with white background?

Reply:
this is required to fulfil UX design requirement